code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -3577,7 +3577,17 @@
 				if(prob(10))
 					to_chat(H,"<span class='warning'>You really don't feel very good.</span>")
 				if(prob(5))
-					H.adjustToxLoss(0.1)
+					switch(pick(1,2,3,4,5))
```

Comment:
It's half memey.
Discount Dans food is hardly lethal. Even in large quantity (200+ units of dans sauce.)
If something is so slow that it's damage has to to be quintupled to show any real effect, it's hardly there.
(also you can heal off the damage easy)